raster/src/main/scala/geotrellis/raster/io/geotiff/tags/TiffTags.scala

```diff
@@ -446,7 +446,13 @@ case class TiffTags(
           i += 1
         }
 
-        Extent(minX, minY, maxX, maxY)
+        // fix an inverted extent, to behave more like GDAL
```

Comment:
do we really want to perform a silent transform or mb we can throw at least a warning?

Reply:
I'm surprised that GDAL allows this but seems like a safe conversion here.